### PR TITLE
[DOCS] Correct cat snapshots API request example

### LINKED ISSUE #50181: cat snapshots API requires repository name
**Elasticsearch version** (`bin/elasticsearch --version`):
`Version: 7.5.0, Build: default/tar/e9ccaed468e2fac2275a3761849cbee64b39519f/2019-11-26T01:06:52.518245Z, JVM: 11.0.5`

**Plugins installed**:
`repository-s3`

**JVM version** (`java -version`):
`OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.5+10, mixed mode)`

**OS version** (`uname -a` if on a Unix-like system):
`Linux appaidf1 3.10.0-957.21.3.el7.x86_64 #1 SMP Fri Jun 14 02:54:29 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux`

**Description of the problem including expected versus actual behavior**:
According to the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-snapshots.html) the `cat snapshots API` should return the content of all repositories, if no specific repository or `_all` is given. Unfortunately neither one works and just produce errors.

Not sure if this is a bug or a documentation issue.

**Steps to reproduce**:
```
GET /_cat/repositories
plgrels7-at-fat s3
```
```
GET /_cat/snapshots
{
    "error": {
        "reason": "Validation Failed: 1: repository is missing;",
        "root_cause": [
            {
                "reason": "Validation Failed: 1: repository is missing;",
                "type": "action_request_validation_exception"
            }
        ],
        "type": "action_request_validation_exception"
    },
    "status": 400
}
```
```
GET /_cat/snapshots/_all
{
    "error": {
        "reason": "[_all] missing",
        "root_cause": [
            {
                "reason": "[_all] missing",
                "type": "repository_missing_exception"
            }
        ],
        "type": "repository_missing_exception"
    },
    "status": 404
}
```
```
GET /_cat/snapshots/plgrels7-at-fat
weekly-2019.11.28-1mpcnufqsgq1fnp2lrp3da SUCCESS 1574977884 21:51:24 1574978190 21:56:30  5.1m 40 40 0 40
weekly-2019.11.29-vohughnnrqmde5dmr69ggw SUCCESS 1575057600 20:00:00 1575057629 20:00:29 29.4s 41 41 0 41
weekly-2019.12.06-ozp2fuuss46awjkjzkfiyg SUCCESS 1575662400 20:00:00 1575662446 20:00:46 46.4s 47 47 0 47
```

### PR DESCRIPTION
**Pull request by jrodewig**
For 7.x and earlier branches, `_cat/repositories` API requests require a
repository name.

This removes an erroneous request example without a repository name
added with a8e0275.

Fixes #50181.

### CODE REVIEW

**Review by original-brownbear — APPROVED** [review]
LGTM thanks @jrodewig !

File: docs/reference/cat/snapshots.asciidoc [hunk 1]
```diff
@@ -23,7 +21,7 @@ more repositories. A snapshot is a backup of an index or running {es} cluster.
 +
 --
 (Optional, string) Comma-separated list of snapshot repositories used to limit
```
Comment (original-brownbear) [inline docs/reference/cat/snapshots.asciidoc]: Sorry just missed this: this isn't optional and not a comma separated list either. It's just a mandatory single string (repository name). Could you fix that as well? :)

Comment (jrodewig) [reply docs/reference/cat/snapshots.asciidoc]: Good catch. Fixed with 022fc9e94d2.

Comment (original-brownbear) [reply docs/reference/cat/snapshots.asciidoc]: >  not a comma separated list either.

Thanks, but this is still missing: It's just the repository name here. No CSV, no wildcards, just the name :)

Comment (jrodewig) [reply docs/reference/cat/snapshots.asciidoc]: 🤦‍♂ Apologies.

Updated with 08bdc76663f.

Comment (original-brownbear) [reply docs/reference/cat/snapshots.asciidoc]: :heart: thanks now it's perfect

**Comment by jrodewig** [comment]
7.x: 99fdea50dd71aef45af1811cece8993da8cad8d8
7.5: d33e5d8bfaaf3df5e5f12168614a484d9dee7a7f
7.4: 65e509adb3c77e1a8d465e97453d79f8c517de35
7.3: 92433370a8c361289c6d45f819093ef86b4849f9

**Pull request merged**